### ISSUE TITLE
Bug: S3 get-bucket-location returning invalid value for LocationConstraint

### ISSUE THREAD

**Issue by vangorra** [localstack/localstack#380]
When running with the node aws-sdk, requesting getBucketLocation results in a improperly formatted value for LocationConstraint.

Node Code:
``` javascript
const s3 = new AWS.S3({
  acessKeyId: 'A',
  secretAccessKey: 'B',
  region: 'us-east-1',
});
s3
  .getBucketLocation({
    Bucket: 'myBucket-us-east-1',
  })
  .promise()
  .then((data) => {
    console.log('RETURNED DATA', data);
  });
```

Returned Value: 
``` 
{ LocationConstraint: '<LocationConstraint xmlns="http://s3.amazonaws.com/doc/2006-03-01/">us-east-1' }
```

Affects:
v0.7.0 through v0.8.1 

Works with: 
v0.6.2

Side note: 
Running get location via the aws cli returns the proper value.
```
aws --endpoint-url=http://localstack:4572 s3api get-bucket-location --bucket myBucket-us-east-1
```
```
{
    "LocationConstraint": "us-east-1"
}
```

**Comment by whummer** [comment]
Thanks for reporting @vangorra , this should be fixed in #386 . Can you please give it a try with the latest version (the Docker image is currently being built: https://travis-ci.org/localstack/localstack/builds/284954545). Thanks

**Comment by vangorra** [comment]
Fixed on on latest. Thanks!
